addon/utils/text.js

```diff
@@ -89,9 +91,8 @@ export function trimLongDataInElement (element) {
   if (textWidth > width) {
     text = trimDataToFit(text, width)
     tooltip = $(element).data('text')
+    $element
```

Comment:
The `frost-select-li` re-renders each time the select is opened (and apparently even each time the hover effect changes in the list), so this isn't really an issue. I'll put in a test case to make sure resizing the select between opening and closing the list continues to work, though.